app/src/main/res/values/styles.xml

```diff
@@ -1,41 +1,87 @@
+        <item name="colorSecondary">@color/secondary</item>
+        <item name="colorOnSecondary">@color/onSecondary</item>
+        <item name="colorSecondaryContainer">@color/secondaryContainer</item>
+        <item name="colorOnSecondaryContainer">@color/onSecondaryContainer</item>
+        <item name="colorTertiary">@color/tertiary</item>
+        <item name="colorOnTertiary">@color/onTertiary</item>
+        <item name="colorTertiaryContainer">@color/tertiaryContainer</item>
+        <item name="colorOnTertiaryContainer">@color/onTertiaryContainer</item>
+        <item name="colorError">@color/error</item>
+        <item name="colorErrorContainer">@color/errorContainer</item>
+        <item name="colorOnError">@color/onError</item>
+        <item name="colorOnErrorContainer">@color/onErrorContainer</item>
+        <item name="android:colorBackground">@color/background</item>
+        <item name="colorOnBackground">@color/onBackground</item>
+        <item name="colorSurface">@color/surface</item>
+        <item name="colorOnSurface">@color/onSurface</item>
+        <item name="colorSurfaceVariant">@color/surfaceVariant</item>
+        <item name="colorOnSurfaceVariant">@color/onSurfaceVariant</item>
+        <item name="colorOutline">@color/outline</item>
+        <item name="colorOnSurfaceInverse">@color/inverseOnSurface</item>
+        <item name="colorSurfaceInverse">@color/inverseSurface</item>
+        <item name="colorPrimaryInverse">@color/inversePrimary</item>
 
-        <item name="android:statusBarColor">?colorPrimary</item>
+        <item name="switchPreferenceCompatStyle">@style/MaterialPreferenceSwitch</item>
+        <item name="android:statusBarColor">@android:color/transparent</item>
+        <item name="android:navigationBarColor">?android:colorBackground</item>
+        <item name="android:windowLightStatusBar">false</item>
     </style>
 
-    <style name="Theme.AdAway.NoActionBar.Red" parent="Theme.MaterialComponents.NoActionBar">
-        <item name="colorPrimary">@color/welcomeBackground</item>
-        <item name="colorPrimaryVariant">@color/primaryVariant</item>
-        <item name="colorOnPrimary">@color/onPrimary</item>
-        <item name="colorSecondary">@color/onPrimary</item>
-        <item name="colorSecondaryVariant">@color/primaryVariant</item>
-        <item name="colorOnSecondary">@color/welcomeBackground</item>
+    <style name="Theme.AdAway.NoActionBar.Monet" parent="Theme.AdAway.NoActionBar">
+        <item name="android:statusBarColor">?attr/colorPrimaryDark</item>
+    </style>
 
-        <item name="android:statusBarColor">?colorOnSecondary</item>
-        <item name="android:navigationBarColor">@color/welcomeBackground</item>
-        <item name="android:windowLightStatusBar">false</item>
+    <style name="MaterialPreferenceSwitch" parent="@style/Preference.SwitchPreferenceCompat.Material">
+        <item name="widgetLayout">@layout/view_preference_switch</item>
     </style>
```

Comment:
Can you detail what the `MaterialPreferenceSwitch` style use for? 🙏

Reply:
That's needed as the preferences are not styled Material Design 3 yet.
By using our custom switch style, our switches look like https://m3.material.io/components/switch/overview instead of the old switches from MD2 (https://m2.material.io/components/switches).